kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -13,7 +13,7 @@ define([
     'bluebird',
     'handlebars',
     'narrativeConfig',
-    'jobCommChannel',
```

Comment:
You can still use `jobCommChannel` here as you're changing the alias in narrative_paths.js. Or just remove the alias from there and go by the `common` directory instead.